internal/clients/team2/agent/actions.go

```diff
@@ -34,8 +51,22 @@ func (a *AgentTwo) DecideAllocation() voting.IdVoteMap {
 }
 
 func (a *AgentTwo) VoteForKickout() map[uuid.UUID]int {
-	// TODO: Vote for the agents with a Social Capital lower than a threshold.
-	return a.BaseBiker.VoteForKickout()
+	VoteMap := make(map[uuid.UUID]int)
+	kickoutThreshold := ChangeBikeSocialCapitalThreshold
+	agentTwoID := a.GetID()
+
+	// check all bikers on the bike but ignore ourselves
+	for _, agent := range a.GetFellowBikers() {
+		if agent.GetID() != agentTwoID {
+			if a.Modules.SocialCapital.SocialCapital[agent.GetID()] < kickoutThreshold {
```

Comment:
What if an AgentId is not in our SC map?

Reply:
Would I be getting average value of SC since its the first time we've rode with this agent?

Reply:
In Go, if you access a key that does not exist in a map, you get a zero value as the value. So the social capital for a non-existant agent will be 0.